src/ioctl.rs

```diff
@@ -173,7 +298,36 @@ pub unsafe fn ioctl_with_ref<F: AsRawFd, T>(fd: &F, req: c_ulong, arg: &T) -> c_
     )
 }
 
-/// Run an ioctl with a mutable reference.
+/// Run an [`ioctl`](http://man7.org/linux/man-pages/man2/ioctl.2.html)
+/// with a mutable reference.
+///
+/// # Arguments
+///
+/// * `fd`: an open file descriptor corresponding to the device that
```

Comment:
How about: "an open file corresponding to the device on which to call the ioctl command"?
Any other suggestions are more than welcomed.

Reply:
That's better. Would you mind to s/ioctl command/ioctl? Not sure if it makes sense. 
I just updated PR with public module descriptor moving into ioctl.rs file.